test/test_katello/test_enabled_report.py

```diff
@@ -1,32 +1,57 @@
 import os
+import imp
 import sys
 import unittest2 as unittest
 from mock import patch
 sys.path.append(os.path.join(os.path.dirname(__file__), '../../src/'))
+from katello import enabled_report
 
 try:
-    from katello import enabled_report
+    imp.find_module('zypp_plugin')
+    ZYPPER = True
 except ImportError:
-    print("Yum wasn't present")
+    ZYPPER = False
 
+try:
+    imp.find_module('yum')
+    YUM = True
+except ImportError:
+    YUM = False
 
 class TestEnabledReport(unittest.TestCase):
-    @unittest.skipIf('yum' not in sys.modules, "Yum not present")
+    @unittest.skipIf(YUM == False, "Yum not present")
+    @patch('katello.enabled_report.YUM', True)
+    @patch('katello.enabled_report.ZYPPER', False)
```

Comment:
Since we check YUM before ZYPPER, I don't think we need to patch ZYPPER since there's an elif